crane/views/crane.py

```diff
@@ -0,0 +1,29 @@
+"""
+Non-public view for use by admins to see a list of repositories served by crane.
+"""
```

Comment:
This comment says that this view is non-public, but nothing in this module seems to restrict access. Is the access restricted elsewhere? If not, it sounds like we need to do this.